components/selector-bar/src/selector-bar-item/selector-bar-item.js

```diff
@@ -29,32 +29,62 @@ export const SelectorBarItem = ({
     className,
     dataTest,
     disabled,
+    displayOnly,
     label,
     noValueMessage,
     open,
     setOpen,
     value,
+    onClearSelectionClick,
 }) => {
     const buttonRef = useRef()
     const Icon = open ? IconChevronUp24 : IconChevronDown24
 
     return (
         <button
             ref={buttonRef}
-            className={cx('selector-bar-item', className)}
+            className={cx(
+                'selector-bar-item',
+                className,
+                !displayOnly ? 'pointer' : ''
```

Comment:
Nitpick: The class name `pointer` includes the style that's supposed to be applied. IMO the class name should be descriptive & declarative, so another name would be more appropriate, maybe `openable` (not sure, but I suppose that's the intention here, right?)? 

Then the style applied when the selector bar item is openable can be changed without having to either change the or have a wrong  class name.

This is super nit picky though, so I'll leave that decision up to you